src/imputation/MoR.py

```diff
@@ -50,6 +58,16 @@ def mor_preprocessing(df, backdata):
         df (pd.DataFrame): full responses for the current year
         backdata (pd.Dataframe): backdata file read in during staging.
     """
+
+    # Catch dataframes which are missing cols 200, 201, formtype, status or manual_trim
+    needed_cols = {"200", "201", "formtype", "status", "manual_trim"}
+    if not needed_cols.issubset(df.columns):
+        missing = needed_cols - set(df.columns)
+        raise ValueError(
+            "The DataFrame must contain columns 200, 201, formtype, status and manual_trim."  # noqa
+            "Missing: {}".format("\n - ".join(missing))
+        )
```

Comment:
Do we still need this in here? I don't think we use manual_trim in MoR anymore?

Reply:
I think this doesn't hurt? Maybe we don't need to check for manual_trim

Can be removed if we are guaranteed that the df will always have these columns, hence making this redundant code.